packages/suite/src/views/view-only/ViewOnlyPromoContent.tsx

```diff
@@ -67,6 +68,22 @@ const LargeDeviceImage = styled(DeviceImage)`
     width: 93px;
 `;
 
+const LargeDeviceImageContainer = styled.div`
+    display: flex;
+    flex: 1 1 auto;
+    align-items: center;
+    justify-content: center;
+    padding: ${spacingsPx.zero} ${spacingsPx.md};
+    width: 100%;
+    overflow: hidden;
+
+    @media (max-width: ${variables.SCREEN_SIZE.SM}) {
+        padding-left: ${spacingsPx.lg};
+        padding-right: ${spacingsPx.lg};
+        padding-bottom: ${spacingsPx.sm};
```

Comment:
Wouldn't be easier to read something like this?

```ts
padding: 0 ${spacingsPx.lg} ${spacingsPx.sm};
```

Reply:
Yep, of course! Just forgot to do this before pushing the code here..